app/_src/policies/introduction.md

```diff
@@ -1,6 +1,651 @@
 ---
 title: Policies
 ---
+{% if_version gte:2.9.x %}
+## What is a policy?
+
+A policy is a set of configuration that will be used to generate the proxy configuration.
+{{ site.mesh_product_name }} combines policies with dataplane configuration to generate the Envoy configuration of a proxy.
+
+## What do policies look like?
+
+Like all [resources](/docs/{{ page.version }}/introduction/concepts#resource) in {{ site.mesh_product_name }}, there are two parts to a policy:
+
+1. The metadata
+2. The spec
+
+### Metadata
+
+Metadata identifies the policies by its `name`, `type` and what `mesh` it's part of:
+
+{% tabs metadata %}
+{% tab metadata Universal %}
+
+```yaml
+type: ExamplePolicy
+name: my-policy-name
+mesh: default
+spec: ... # spec data specific to the policy kind
+```
+
+{% endtab %}
+{% tab metadata Kubernetes %}
+
+In Kubernetes all our policies are implemented as [custom resource definitions (CRD)](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) in the group `kuma.io/v1alpha1`.
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: ExamplePolicy
+metadata:
+  name: my-policy-name
+  namespace: {{ site.mesh_namespace }}
+spec: ... # spec data specific to the policy kind
+```
+
+By default the policy is created in the `default` mesh.
+You can specify the mesh by using the `kuma.io/mesh` label.
+
+For example:
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: ExamplePolicy
+metadata:
+  name: my-policy-name
+  namespace: {{ site.mesh_namespace }}
+  labels:
+    kuma.io/mesh: "my-mesh"
+spec: ... # spec data specific to the policy kind
+```
+
+{% warning %}
+Policies are namespaced scope and currently the namespace must be the one the control-plane is running in `{{site.mesh_namespace}}` by default.
+{% endwarning %}
+
+{% endtab %}
+{% endtabs %}
+
+### Spec
+
+The `spec` field contains the actual configuration of the policy.
+
+Some policies apply to only a subset of the configuration of the proxy.
+
+- **Inbound policies** apply only to incoming traffic. The `spec.from[].targetRef` field defines the subset of clients that are going to be impacted by this policy.
+- **Outbound policies** apply only to outgoing traffic. The `spec.to[].targetRef` field defines the outbounds that are going to be impacted by this policy
```

Comment:
"outbounds" as a concept IMO isn't widely defined, we should use "destinations" or similar, like "clients" in the line above.

Reply:
Yeah I'm lukewarm on outbounds/inbounds but I'm also not mega fan of clients vs destinations as I find not them not mega clear to me on which proxy and where they apply.
I think the GUI shows inbound/outbounds and the data-planes have inbound and outbounds. So I really think inbound/outbound policy is clearer...

Reply:
The previous line uses clients though. Maybe just "outbounds, or traffic destinations for this proxy" something like that?

Reply:
I think in the current state on "inbounds" it's actually clients though there's a lot to be done there